source/boundary_temperature/interface.cc

```diff
@@ -272,17 +275,6 @@ namespace aspect
                           "will be used to append the listed temperature models onto "
                           "the previous models. If only one operator is given, "
                           "the same operator is applied to all models.");
```

Comment:
I know this doesn't have anything to do with your pull request, but can you please add a sentence here (and in other places where we declare these operators), that this list of operators has to be exactly as long as the list of plugins (if it doesn't have just one entry), and that the order has to be the same as in the 'List of model names' because each operator will be applied to a plugin according to the order they're given in.